.github/workflows/swift_package_test.yml

```diff
@@ -450,7 +450,7 @@ jobs:
           Invoke-Program swift test --version
           Invoke-Program cd $Source
           ${{ inputs.windows_pre_build_command }}
-          Invoke-Program ${{ inputs.windows_build_command }} ${{ (contains(matrix.swift_version, 'nightly') && inputs.swift_nightly_flags) || inputs.swift_flags }}
```

Comment:
We could also leave this in, I am slightly concerned that `Invoke-Program` isn't being used in downstream overrides today.

Reply:
I don’t think we should prefix it with `Invoke-Program`. Otherwise, you might learn that the command fails if you only have a single one in your `windows_build_commands` and are then surprised that this behavior doesn’t hold up when you add a second command to the script. 

Instead, what I would suggest is that we go through the existing use cases of this workflow with `windows_build_command` and fix them. There’s only a handful and many of them are forks: https://github.com/search?q=swiftlang%2Fgithub-workflows%2F.github%2Fworkflows%2Fswift_package_test.yml%40main+windows_build_command&type=code

Reply:
Yeah, this is why I removed it. I'll go through them

Reply:
https://github.com/swiftlang/swift-syntax/pull/3140
https://github.com/swiftlang/swift-build/pull/746
https://github.com/swiftlang/vscode-swift/pull/1800
https://github.com/swiftlang/swift-foundation-icu/pull/77
https://github.com/swiftlang/swift-format/pull/1060
https://github.com/swiftlang/swift-toolchain-sqlite/pull/20